### PR TITLE
feat: video playback error ui implemented

### PR DESCRIPTION
**Pull request by AryanCypherock**
- Error will be displayed when trying to play videos offline

### CODE REVIEW

File: packages/ui/src/components/atoms/VideoPlaybackError.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+> = {
+  default: css<VideoPlaybackErrorProps>`
+    width: 720px;
+    height: 405px;
+  `,
+  big: css<VideoPlaybackErrorProps>`
+    width: 900px;
+    height: 508px;
+  `,
+};
```
Comment (TejasvOnly) [inline packages/ui/src/components/atoms/VideoPlaybackError.tsx]: Why do we need to specify the size it should take the size of it's parent

File: packages/ui/src/components/atoms/VideoPlaybackError.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  ...props
+}) => (
+  <StyledVideoPlaybackError {...props}>
+    <StyledInfo>
+      <Image
+        src={informationWhiteIcon}
+        alt="info icon"
+        $width={40}
+        $height={42}
+      />
```
Comment (TejasvOnly) [inline packages/ui/src/components/atoms/VideoPlaybackError.tsx]: Use svg icon component

File: packages/cysync-core/src/components/CustomReactPlayer.tsx
Comment (TejasvOnly) [inline packages/cysync-core/src/components/CustomReactPlayer.tsx:1]: Name this `VideoPlayer` and extend `ReactPlayer` props so we can seamlessy replace this with react player everywhere

File: packages/cysync-core/src/components/CustomReactPlayer.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+const sizeMap = {
+  default: {
+    width: '720px',
+    height: '405px',
+  },
+  big: {
+    width: '900px',
+    height: '508px',
+  },
+};
```
Comment (TejasvOnly) [inline packages/cysync-core/src/components/CustomReactPlayer.tsx]: No need for this use props for `ReactPlayer`

File: packages/cysync-core-constants/src/i18n/lang/en.json [hunk 4]
```diff
@@ -1114,7 +1114,8 @@
         "subtext": "Reconnect the device and try again and if the problem persists, contact Cypherock support for assistance"
       }
     },
-    "default": "Some internal error occurred"
+    "default": "Some internal error occurred",
+    "videoPlaybackError": "No Internet connection. Please check your connection and try again"
```
Comment (TejasvOnly) [inline packages/cysync-core-constants/src/i18n/lang/en.json]: We already have `SER_0001`, consult with akshit to see if we really need a different message for this

File: packages/ui/src/components/atoms/VideoPlaybackError.tsx
Comment (TejasvOnly) [inline packages/ui/src/components/atoms/VideoPlaybackError.tsx:1]: Do we really need this component?

**Review by TejasvOnly — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/atoms/VideoPlaybackError.tsx (hunk 2 above)
Comment (AryanCypherock) [reply packages/ui/src/components/atoms/VideoPlaybackError.tsx]: The svg doesn't scale properly for this icon, that's why using the image

File: packages/cysync-core-constants/src/i18n/lang/en.json (hunk 4 above)
Comment (AryanCypherock) [reply packages/cysync-core-constants/src/i18n/lang/en.json]: Akshit told me to use the new message

File: packages/cysync-core/src/dialogs/Inheritance/GoldPlanPurchase/Dialogs/Instructions.tsx [hunk 5]
```diff
@@ -43,10 +43,11 @@ export const Instructions = () => {
           <LangDisplay text={strings.instructions.video.title} />
         </Typography>
       </Container>
-      <ReactPlayer
-        url={constants.inheritance.goldPlanPurchaseTutorialLink}
+      <VideoPlayer
+        url={constants.inheritance.silverPlanPurchaseTutorialLink}
```
Comment (TejasvOnly) [inline packages/cysync-core/src/dialogs/Inheritance/GoldPlanPurchase/Dialogs/Instructions.tsx]:
```suggestion
        url={constants.inheritance.goldPlanPurchaseTutorialLink}
```

File: packages/cysync-core/src/dialogs/Inheritance/GoldPlanPurchase/Dialogs/Message/ExecutorMessageTutorial.tsx [hunk 6]
```diff
@@ -46,10 +46,12 @@ export const ExecutorMessageTutorial = () => {
           <LangDisplay text={strings.tutorial.subtext} />
         </Typography>
       </Container>
-      <ReactPlayer
+
+      <VideoPlayer
         url={constants.inheritance.silverPlanPurchaseTutorialLink}
         width="720px"
         height="405px"
+        onRetry={onRetry}
```
Comment (TejasvOnly) [inline packages/cysync-core/src/dialogs/Inheritance/GoldPlanPurchase/Dialogs/Message/ExecutorMessageTutorial.tsx:55]: Retry shouldn't restart the complete flow, only reload the current page.

**Review by TejasvOnly — CHANGES_REQUESTED** [review]
Have this been tested locally, try to test while the interenet is disabled on the host machine not only from the browser level. may need to add some sort of timeout.

**Review by TejasvOnly — CHANGES_REQUESTED** [review]
Update the success page in estate recovery as well, and resovle the conflicts

**Review by TejasvOnly — APPROVED** [review]

**Pull request merged**